### ISSUE TITLE
Grid Row is not screen reader friendly

### ISSUE THREAD

**Issue by dmitri-suvorov-sumup** [sumup-oss/circuit-ui#564]
The `Row` component inserts dots into the DOM before and after the container. These dots are not visible for the user but are being recognized by the screen reader. This results in audio noise.

## Sample layout
```jsx
<StyledRow>
  <StyledCol>Header</StyledCol>
</StyledRow>
<StyledRow>
  <StyledCol>Content</StyledCol>
</StyledRow>
<StyledRow>
  <StyledCol>Footer</StyledCol>
</StyledRow>
```

![row](https://user-images.githubusercontent.com/53574544/78983342-b8f40a00-7b2c-11ea-849b-3b85f939b8fe.png)

## Screen reader output (VoiceOver on macOS Catalina)
![period4](https://user-images.githubusercontent.com/53574544/78983124-3cf9c200-7b2c-11ea-847d-5a484a08c98e.gif)

**Comment by connor-baer** [comment]
That's...interesting. Thank you for reporting this! I love seeing others using/testing with screen readers! 🙌

Would you be able to contribute a fix for this?

**Comment by dmitri-suvorov-sumup** [comment]
Yes, I'm on it
